.github/workflows/CreateRelease.yml

```diff
@@ -78,7 +78,7 @@ jobs:
   publish:
     # see https://github.com/orgs/community/discussions/26286#discussioncomment-3251208 for why we need to check the ref
     if: ${{ contains(github.ref, 'refs/heads/release/') }} ||  ${{ github.ref=='refs/heads/main' }}
-    runs-on: windows-2022
+    runs-on: ubuntu-latest
```

Comment:
Can you use our KVM runner?
we pre-install the some cargo components

Reply:
fixed